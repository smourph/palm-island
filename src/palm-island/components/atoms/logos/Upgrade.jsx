import React from 'react';
import upgrade from '@internal/resources/images/logo-upgrade.png';
import './Logo.scss';

export const Upgrade = () => (
  <span className="logo logo-common logo-upgrade">
    <img src={upgrade} alt="upgrade"/>
  </span>
);

export default Upgrade;
