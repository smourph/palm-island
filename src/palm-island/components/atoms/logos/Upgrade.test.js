import React from 'react';
import {render, screen} from '@testing-library/react';
import Upgrade from './Upgrade';

it('should render snapshot', () => {
  const {asFragment} = render(<Upgrade/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render upgrade', () => {
  render(<Upgrade/>);

  const icon = screen.getByAltText('upgrade');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-upgrade.png');
});
