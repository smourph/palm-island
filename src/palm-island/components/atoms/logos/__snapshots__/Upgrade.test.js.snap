// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-common logo-upgrade"
>
  <img
    alt="upgrade"
    src="logo-upgrade.png"
  />
</span>
`;
